Mathlib/Analysis/NormedSpace/FunctionSeries.lean

```diff
@@ -44,6 +44,46 @@ theorem tendstoUniformlyOn_tsum_nat {f : ℕ → β → F} {u : ℕ → ℝ} (hu
       s :=
   fun v hv => tendsto_finset_range.eventually (tendstoUniformlyOn_tsum hu hfu v hv)
 
+/-- An infinite sum of functions with eventually summable sup norm is the uniform limit of its
+partial sums. Version relative to a set, with general index set. -/
+theorem tendstoUniformlyOn_tsum_eventually {ι : Type*} {f : ι → β → F} {u : ι → ℝ}
+    (hu : Summable u) {s : Set β}
+    (hfu : ∃ a : Finset ι, ∀ᵉ (b ⊇ a) (x ∈ s) (n ∉ b), ‖f n x‖ ≤ u n) :
```

Comment:
Isn't this equivalent to:
```suggestion
theorem tendstoUniformlyOn_tsum_of_eventually {ι : Type*} {f : ι → β → F} {u : ι → ℝ}
    (hu : Summable u) {s : Set β}
    (hfu : ∀ᶠ n in cofinite ι, ∀ x ∈ s, ‖f n x‖ ≤ u n) :
```
which is an *actual* `Filter.Eventually` statement? I think this is nicer, the condition should be easier to verify for downstream users, and I bet the proof can be simplified as a result (although I'm not sure about this last claim). Note also the name should be changed.